### ISSUE TITLE
lighten

### ISSUE THREAD

**Issue by TexRexTex** [cabaletta/baritone#1112]
## Describe your suggestion
`#lighten <lightsource>`
this command will check for the best place to put a light source and put it there so that mobs will not spawn when you are mining

## Settings
MoveLightSource default true
this will find light sources that are not in a good position and move them to a better one.

## Context
This would be a great feature to baritone because it would make it much easier to automine without being killed by hostile mobs.

## Final checklist
- [tick] I have not used any OwO's or UwU's in this issue.

**Comment by usereewrjrjeejr** [comment]
Just turn on full bright

**Comment by TexRexTex** [comment]
That will not stop mobs spawning thou.

**Comment by Azuranv2682** [comment]
This needs to be a feature

**Comment by SWEDCODER** [comment]
Yeah maybe not a full feature for #mine cause it slows down a lot. but it could be a boolean setting. The other case I can see is just a feature to lighten a full area corner to corner, it could even be in #sel.
